ortho_seq_code/orthogonal_polynomial.py

```diff
@@ -930,126 +924,26 @@ def orthogonal_polynomial(
     print("Trait values estimated from regressions")
     print(Fest)
 
-    ## Graph of regression
-    # Flatten data
-    rFon1D_flat = list(rFon1D.flatten())
-    if any(i != 0 for i in rFon1D_flat):
-        data_null = np.where(
-            np.array(rFon1D_flat) == float(0), float("nan"), rFon1D_flat
-        )
-
-        # Constants/constant arrays
-        ind = np.arange(sites)  # x-axis
-        num_dm = np.arange(dm)
-        width = 1 / sites
-        s = sites * dm
-
-        # Re-vectorization with null values
-        dim_num = dict()
-        for i in ind:
-            dim_num[i] = [data_null[j] for j in np.arange(dm * i, dm * i + dm)]
-        # some_dim = [data_array_flat[i], i for i in range(0, 160, 4)]
-
-        # Remove all null data
-        dim_na = dict()
-        dim_loc = dict()
-        for i in ind:
-            dim_na[i] = np.array(dim_num[i])[np.array(np.isnan(dim_num[i])) == False]
-            dim_loc[i] = np.arange(len(dim_num[i]))[
-                np.array(np.isnan(dim_num[i])) == False
-            ]
-        # Color dictionary with corresponding letters
-
-        dim_aa = dict()
-
-        for i in num_dm:
-            dim_aa[i] = [data_null[j] for j in range(i, s, dm)]
+    ## Bar plot of regression
 
-        col_len = len(colors)
-        alpb_d = dict()
-        if alphbt_input is None:
-            for i in num_dm:
-                if any(i != 0 and i for i in dim_aa[i]):
-                    alpb_d[i] = colors[i % col_len]
-                    alpb_d[alphabets[i]] = alpb_d.pop(i)
-        else:
-            for i in num_dm:
-                if any(i != 0 and i for i in dim_aa[i]):
-                    alpb_d[i] = colors[i % col_len]
-                    alpb_d[custom_aa[i]] = alpb_d.pop(i)
-
-        # Creating plots
-        fig, ax = plt.subplots()
-        dim = dict()
-        pi = dict()
-        for i in range(sites + 1):
-            ax.axvline(i, color="lightgray", linewidth=0.8, zorder=0)
-        for i in ind:
-            if len(dim_na[i]) == 0:
-                ln = 1
-            else:
-                ln = 1 / len(dim_na[i])
-            rn = np.arange(1 / ln)
-            pi[i] = ax.bar(
-                x=i + np.array([j for j in rn]) * ln,
-                height=[j for j in dim_na[i]],
-                width=ln,
-                align="edge",
-                color=[colors[i % col_len] for i in list(dim_loc[i])],
-                edgecolor="black",
-                zorder=3,
+    if custom_aa is not None:
+        if pheno_name is not None:
+            rFon1D_o = rf1d(
+                rFon1D, custom_aa, molecule=molecule, custom=True, phenotype=pheno_name
```

Comment:
do we need the custom=True here? because when running this, we get this error:  
```
File "/Users/saba.nafees/miniconda3/envs/ortho_seqs/lib/python3.9/site-packages/ortho_seqs-1.0.1-py3.9.egg/ortho_seq_code/orthogonal_polynomial.py", line 931, in orthogonal_polynomial
    rFon1D_o = rf1d(
TypeError: __init__() got an unexpected keyword argument 'custom'
```

Reply:
That was some code I forgot to remove when I removed the custom parameter from the rf1d class file, it should be fixed now :)